src/Sylius/Behat/Page/Admin/Country/CreatePage.php

```diff
@@ -21,4 +21,17 @@
 class CreatePage extends BaseCreatePage implements CreatePageInterface
 {
     use ChoosesName;
+
+    /**
+     * {@inheritdoc}
+     */
+    public function fillProvinceNameAndCode($name, $code)
+    {
+        $this->getDocument()->clickLink('Add province');
+
+        $provinces = $this->getDocument()->find('css', '#sylius_country_provinces');
+
+        $provinces->fillField('sylius_country_provinces_0_name', $name);
+        $provinces->fillField('sylius_country_provinces_0_code', $code);
```

Comment:
I would rather pick first element of provinces collection and fill its fields, to make this method independent from generated id's.